test/integration/008_retry_failed_test/test_retry_failed.py

```diff
@@ -0,0 +1,14 @@
+from test.integration.base import DBTIntegrationTest
+
+
+class RetryFailedTest(DBTIntegrationTest):
+    @property
+    def models(self):
+        return "test/integration/008_retry_failed_test/models"
+
+    def tests_retry_failed(self):
+        try:
+            self.run_dbt(["run"])
+        except InvocationError:
```

Comment:
This is janky. I am expecting the first run to fail, so I can get the failed results into `run_results.json`
And it's causing tox to fail, except that the command actually works!

Reply:
I actually think this makes sense! Weird that tox is still failing with this though :(